### ISSUE TITLE
Problem with Multipolygones

### ISSUE THREAD

**Issue by kfhenning** [timokorkalainen/Unity-GeoJSONObject#3]
Hey Time,
nice library and really easy to use! Thanks!
However, I have a problem parsing multipolygones. The coordinates are not parsed and it returns always coordinates[0] = (0,0). 
I've tried to parse the example from https://tools.ietf.org/html/rfc7946#section-3.1.7 (p. 10).
I looked through the code and it seems that there is missing an ArrayArrayArrayGeometryObject parser as Polygon and Multipolygon are treated equaly. Or did I get something wrong?

Thanks!
Best regards,
Kai

**Comment by lvictorino** [comment]
Hey @kfhenning, I came to the same conclusion about `ArrayArrayArrayGeometryObject`. It's definitely missing. Did you make a pull request?

**Comment by EgilSandfeld** [comment]
Yes I have the same issue, where multipolygon coordinates end up being 0,0. It needs to be able to parse the sub-arrays for multi polygons, instead of treating the "coordinates" as final lat/lngs.

**Comment by Torantulino** [comment]
Has anyone solved this yet?

**Comment by Torantulino** [comment]
Ran into this problem myself so I've given it a shot!

**Comment by wilcotomassen** [comment]
#4 from @Torantulino solved the same issue with multipolygons I had, thanks!
